database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Build/82_DML_LEASE_TESTING.sql

```diff
@@ -0,0 +1,55 @@
+
+-- Insert people
+INSERT INTO PIMS_PERSON (ADDRESS_ID, SURNAME, FIRST_NAME, MIDDLE_NAMES, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Hollywood%'), N'Smith',  N'Bob',    N'Billy',        CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE 'PO Box%'),     N'Wilson', N'Volley', N'Ball',         CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '123 Main%'),   N'Cheese', N'Stinky', N'',             CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Souris%'),    N'Mouse',  N'Minnie', N'Nacho Cheese', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data');
+
+-- Insert contact information for people
+INSERT INTO PIMS_CONTACT_METHOD (PERSON_ID, CONTACT_METHOD_TYPE_CODE, CONTACT_METHOD_VALUE, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  ((SELECT PERSON_ID FROM PIMS_PERSON WHERE SURNAME = 'Smith'),  N'PERSMOBIL', N'6049934251', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT PERSON_ID FROM PIMS_PERSON WHERE SURNAME = 'Wilson'), N'PERSMOBIL', N'2509977251', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT PERSON_ID FROM PIMS_PERSON WHERE SURNAME = 'Cheese'), N'PERSMOBIL', N'6049983251', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  ((SELECT PERSON_ID FROM PIMS_PERSON WHERE SURNAME = 'Mouse'),  N'PERSMOBIL', N'7785761234', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data');
+
+-- Insert contact information for organizations
+INSERT INTO PIMS_CONTACT_METHOD (ORGANIZATION_ID, CONTACT_METHOD_TYPE_CODE, CONTACT_METHOD_VALUE, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  ((SELECT ORGANIZATION_ID FROM PIMS_ORGANIZATION WHERE ORGANIZATION_IDENTIFIER = 'BOB123'),   N'WORKMOBIL', N'6049934251', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
```

Comment:
This isn't compatible with our current CONTACT_METHOD_TYPE population script provided previously: 
PRINT N'Adding [PIMS_CONTACT_METHOD_TYPE]'

INSERT INTO PIMS_CONTACT_METHOD_TYPE (CONTACT_METHOD_TYPE_CODE, DESCRIPTION)
VALUES
  (N'FAX', N'Facsimile machine'),
  (N'PERSPHONE', N'Personal phone'),
  (N'WORKPHONE', N'Work phone'),
  (N'PERSEMAIL', N'Personal email'),
  (N'WORKEMAIL', N'Work email');

Reply:
Got ahead of the build.  I'll amend the script and the data to the correct value for your build.

Reply:
Changed in my scripts.  I'll rebuild and let you know.